pom.xml

```diff
@@ -533,6 +528,33 @@
 		<!-- Handles deploying snapshots and releases. -->
 		<profile>
 			<id>release</id>
+			<activation>
+				<property>
+					<name>release</name>
+				</property>
+			</activation>
+			<modules>
+				<!-- Support Modules-->
+				<module>spring-cloud-gcp-starters</module>
+				<module>spring-cloud-gcp-dependencies</module>
+
+				<!-- Production Libraries -->
+				<module>spring-cloud-gcp-autoconfigure</module>
+				<module>spring-cloud-gcp-bigquery</module>
+				<module>spring-cloud-gcp-cloudfoundry</module>
+				<module>spring-cloud-gcp-core</module>
+				<module>spring-cloud-gcp-data-datastore</module>
+				<module>spring-cloud-gcp-data-firestore</module>
+				<module>spring-cloud-gcp-data-spanner</module>
+				<module>spring-cloud-gcp-logging</module>
+				<module>spring-cloud-gcp-pubsub</module>
+				<module>spring-cloud-gcp-pubsub-stream-binder</module>
+				<module>spring-cloud-gcp-security-iap</module>
+				<module>spring-cloud-gcp-storage</module>
+				<module>spring-cloud-gcp-secretmanager</module>
+				<module>spring-cloud-gcp-security-firebase</module>
+				<module>spring-cloud-gcp-vision</module>
```

Comment:
Since we no longer have a maven.config, would it be easier to have a `<activeByDefault>` section with all production modules?

Reply:
I ended up putting the maven.config back in. I originally removed it trying to get `activeByDefault` working but no luck.